### ISSUE TITLE
Mouth shapes to blendshapes

### ISSUE THREAD

**Issue by hrdisaac** [DanielSWolf/rhubarb-lip-sync#118]
Is there a mapping table anywhere which can be used to map the identified mouth shape by Rhubarb to the ARKit blendshaes (e.g. jawOpen and mouthPucker)?

https://developer.apple.com/documentation/arkit/arfaceanchor/blendshapelocation

**Comment by DanielSWolf** [comment]
No, there is no such table. But looking at your link, mapping the nine mouth shapes shouldn't be hard.
